scripts/test_case_2.6.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# test case 2.3
+#
+# create an adi from another adi
+# server IP:Port needed unless defaulting to localhost
+#
+# set cli command and see if it exists
+#
+export cli=../cmd/cli/cli
```

Comment:
probably should ensure the cli is built, maybe?  

go run ../cmd/cli/main.go might be an option, but that will be very slow if doing loops.